### ISSUE TITLE
npm run dev gives digital envelope routines::unsupported

### ISSUE THREAD

**Issue by abulka** [JoseGoncalves/vue-jointjs#3]
I followed the install instructions and `npm run dev` gives me `digital envelope routines::unsupported`. Any ideas?

```
$ npm run dev

> vue-jointjs@1.0.0 dev
> webpack-dev-server --mode development --open --hot

ℹ ｢wds｣: Project is running at http://localhost:8081/
ℹ ｢wds｣: webpack output is served from /
ℹ ｢wds｣: Content not from webpack is served from /Users/andy/Devel/test-projects/vue-jointjs
Error: error:0308010C:digital envelope routines::unsupported
    at new Hash (node:internal/crypto/hash:71:19)
    at Object.createHash (node:crypto:133:10)
    at module.exports (/Users/andy/Devel/test-projects/vue-jointjs/node_modules/webpack/lib/util/createHash.js:135:53)
    at NormalModule._initBuildHash (/Users/andy/Devel/test-projects/vue-jointjs/node_modules/webpack/lib/NormalModule.js:417:16)
    at handleParseError (/Users/andy/Devel/test-projects/vue-jointjs/node_modules/webpack/lib/NormalModule.js:471:10)
    at /Users/andy/Devel/test-projects/vue-jointjs/node_modules/webpack/lib/NormalModule.js:503:5
    at /Users/andy/Devel/test-projects/vue-jointjs/node_modules/webpack/lib/NormalModule.js:358:12
    at /Users/andy/Devel/test-projects/vue-jointjs/node_modules/loader-runner/lib/LoaderRunner.js:373:3
    at iterateNormalLoaders (/Users/andy/Devel/test-projects/vue-jointjs/node_modules/loader-runner/lib/LoaderRunner.js:214:10)
    at iterateNormalLoaders (/Users/andy/Devel/test-projects/vue-jointjs/node_modules/loader-runner/lib/LoaderRunner.js:221:10)
    at /Users/andy/Devel/test-projects/vue-jointjs/node_modules/loader-runner/lib/LoaderRunner.js:236:3
    at runSyncOrAsync (/Users/andy/Devel/test-projects/vue-jointjs/node_modules/loader-runner/lib/LoaderRunner.js:130:11)
    at iterateNormalLoaders (/Users/andy/Devel/test-projects/vue-jointjs/node_modules/loader-runner/lib/LoaderRunner.js:232:2)
    at Array.<anonymous> (/Users/andy/Devel/test-projects/vue-jointjs/node_modules/loader-runner/lib/LoaderRunner.js:205:4)
    at Storage.finished (/Users/andy/Devel/test-projects/vue-jointjs/node_modules/enhanced-resolve/lib/CachedInputFileSystem.js:55:16)
    at /Users/andy/Devel/test-projects/vue-jointjs/node_modules/enhanced-resolve/lib/CachedInputFileSystem.js:91:9
ℹ ｢wdm｣: wait until bundle finished: /
/Users/andy/Devel/test-projects/vue-jointjs/node_modules/loader-runner/lib/LoaderRunner.js:133
		if(isError) throw e;
		            ^

Error: error:0308010C:digital envelope routines::unsupported
    at new Hash (node:internal/crypto/hash:71:19)
    at Object.createHash (node:crypto:133:10)
    at module.exports (/Users/andy/Devel/test-projects/vue-jointjs/node_modules/webpack/lib/util/createHash.js:135:53)
    at NormalModule._initBuildHash (/Users/andy/Devel/test-projects/vue-jointjs/node_modules/webpack/lib/NormalModule.js:417:16)
    at handleParseError (/Users/andy/Devel/test-projects/vue-jointjs/node_modules/webpack/lib/NormalModule.js:471:10)
    at /Users/andy/Devel/test-projects/vue-jointjs/node_modules/webpack/lib/NormalModule.js:503:5
    at /Users/andy/Devel/test-projects/vue-jointjs/node_modules/webpack/lib/NormalModule.js:358:12
    at /Users/andy/Devel/test-projects/vue-jointjs/node_modules/loader-runner/lib/LoaderRunner.js:373:3
    at iterateNormalLoaders (/Users/andy/Devel/test-projects/vue-jointjs/node_modules/loader-runner/lib/LoaderRunner.js:214:10)
    at iterateNormalLoaders (/Users/andy/Devel/test-projects/vue-jointjs/node_modules/loader-runner/lib/LoaderRunner.js:221:10)
    at /Users/andy/Devel/test-projects/vue-jointjs/node_modules/loader-runner/lib/LoaderRunner.js:236:3
    at context.callback (/Users/andy/Devel/test-projects/vue-jointjs/node_modules/loader-runner/lib/LoaderRunner.js:111:13)
    at Object.loader (/Users/andy/Devel/test-projects/vue-jointjs/node_modules/eslint-loader/dist/index.js:27:8)
    at LOADER_EXECUTION (/Users/andy/Devel/test-projects/vue-jointjs/node_modules/loader-runner/lib/LoaderRunner.js:119:14)
    at runSyncOrAsync (/Users/andy/Devel/test-projects/vue-jointjs/node_modules/loader-runner/lib/LoaderRunner.js:120:4)
    at iterateNormalLoaders (/Users/andy/Devel/test-projects/vue-jointjs/node_modules/loader-runner/lib/LoaderRunner.js:232:2) {
  opensslErrorStack: [ 'error:03000086:digital envelope routines::initialization error' ],
  library: 'digital envelope routines',
  reason: 'unsupported',
  code: 'ERR_OSSL_EVP_UNSUPPORTED'
}

Node.js v18.15.0
```

**Comment by JoseGoncalves** [comment]
@abulka Hi, thanks for reporting. This is some issue caused by using old webpack dependencies... I'm going to update this project to vite in order to solve it.

**Comment by JoseGoncalves** [comment]
Fixed in commit 30b718c5de3cb8b94ac5932c357b5365dc5df337.
